pom.xml

```diff
@@ -426,12 +432,14 @@
             <dependency>
                 <groupId>io.airlift</groupId>
                 <artifactId>concurrent</artifactId>
-                <version>202</version>
+                <!-- 207 requires Java 11 -->
+                <version>206</version>
```

Comment:
Would it make sense to have a version that is different based on the target java version we are compiling for?

Reply:
Currently, we only produce builds that are binary compatible with 1.8. I would suggest to drop 1.8 compatibility and switch to 11 compatibility as part of our 0.24.x release.